challenges/README.md

```diff
@@ -51,7 +51,18 @@ same as the users object on Users Api
     }
 ]
 
+
 ```
+- **Error Response:**
+  - **Code:** 503
+    - **Content:** 
+    ```
+    {
+        "statusCode": 503,
+        "error": "Service Unavailable",
+        "message": "Something went wrong. Please try again or contact admin"
```

Comment:
Can we make it 2 space indentation?

Reply:
Should we follow 2 space indentation for all JSON code blocks or only error responses?